CMake/packages/Findreportinglibsonata.cmake

```diff
@@ -0,0 +1,41 @@
+# =============================================================================
```

Comment:
Didn't we say we will have just libsonata that will contain everything? In that case I would rename this to `Findlibsonata.cmake` (and there might be some already exist?).

Reply:
What do you mean with that it might already exist? I think we should separate them anyway, maybe Findsonatareports.cmake as you mention in the next comment